src/app/shared/Styles.elm

```diff
@@ -0,0 +1,23 @@
+module Shared.Styles exposing (..)
+
+import Css exposing (..)
+import Html.CssHelpers
+
+
+type CssClasses
+    = Antialiased
+    | Header
+
+
+sharedNamespace =
+    Html.CssHelpers.withNamespace "shared"
+
+
+css : Stylesheet
+css =
+    stylesheet
+        [ class Antialiased
```

Comment:
It might be duplicate yes, `src/index.css` was part of create-elm-app but then we switched to elm-css instead